packages/frontend/src/components/header/HeaderRoutes.tsx

```diff
@@ -19,16 +19,13 @@ const useStyles = makeStyles((theme: any) => ({
 
 const HeaderRoutes: FC = () => {
   const { pathname, search } = useLocation()
-  const history = useHistory()
+  const navigate = useNavigate()
   const styles = useStyles()
   const theme = useTheme()
 
   const handleChange = (event: ChangeEvent<{}>, newValue: string) => {
     event.preventDefault()
-    history.push({
-      pathname: newValue,
-      search,
-    })
+    navigate(newValue + search)
```

Comment:
suggestion: use string interpolation instead

```suggestion
    navigate(`${newValue}${search}`)
```